### ISSUE TITLE
Modelica.Mechanics.Translational.Components.ElastoGap c and d parameters need types

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#873]
**Reported by jriel on 12 Oct 2012 18:50 UTC**
The c and d parameters are specified as
```
  parameter Real c(final unit="N/m", final min=0, start=1) "Spring constant";
  parameter Real d(final unit="N/ (m/s)", final min=0, start=1) "Damping constant";
```
Rather than **Real**, they should be declared as
```
  parameter SI.TranslationalSpringConstant c(final min=0, start = 1) "Spring constant";
  parameter SI.TranslationalDampingConstant d(final min=0, start = 1) "Damping constant";
```

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/873

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 12 Oct 2012 19:04 UTC**
FIXED in f34cbccdea0df6b70a9b64ec4bb5fa1fcb77f406

**Comment by modelica-trac-importer** [comment]
**Changelog modified by dietmarw on 12 Oct 2012 19:04 UTC**
Enhanced `ElastoGap` parameters by usage of SIunits.
